src/test/java/edu/cornell/kfs/fp/batch/xml/fixture/AccountingXmlDocumentEntryFixture.java

```diff
@@ -811,6 +840,13 @@ public AccountingDocument toAccountingDocument(String documentNumber) {
             accountingDocument.setPostingYear(postingFiscalYear);
         }
         
+        if (accountingDocumentClass == PreEncumbranceDocument.class) {

```

Comment:
For consistency with similar unit test code in this enum, I would suggest moving this PE-doc-setup block into its own method. See the `addDvDetailsToDocumentIfNecessary()` and `addAuxiliaryVoucherSettingsToDocumentIfNecessary()` methods for examples.

Reply:
ok